packages/director/src/screenshots/s3/s3.ts

```diff
@@ -1,78 +1,64 @@
-import { AssetUploadInstruction } from '@sorry-cypress/common';
-import aws from 'aws-sdk';
-import { sanitizeS3KeyPrefix } from '../utils/';
+import { fromEnv } from "@aws-sdk/credential-providers";
+import {AssetUploadInstruction} from '@sorry-cypress/common';
+import {HttpRequest} from "@aws-sdk/protocol-http";
+import {S3RequestPresigner} from "@aws-sdk/s3-request-presigner";
+import {parseUrl} from "@aws-sdk/url-parser";
+import {formatUrl} from "@aws-sdk/util-format-url";
+import {Hash} from "@aws-sdk/hash-node";
+import {sanitizeS3KeyPrefix} from '../utils/';
 import {
-  S3_ACL,
-  S3_BUCKET,
-  S3_FORCE_PATH_STYLE,
-  S3_IMAGE_KEY_PREFIX,
-  S3_READ_URL_PREFIX,
-  S3_REGION,
-  S3_VIDEO_KEY_PREFIX,
-  UPLOAD_EXPIRY_SECONDS,
+    S3_URL,
+    S3_BUCKET,
+    S3_IMAGE_KEY_PREFIX,
+    S3_REGION,
+    S3_VIDEO_KEY_PREFIX,
+    UPLOAD_EXPIRY_SECONDS
 } from './config';
-import { S3SignedUploadResult } from './types';
-
-const BUCKET_URL = S3_FORCE_PATH_STYLE
-  ? `https://s3.amazonaws.com/${S3_BUCKET}`
-  : `https://${S3_BUCKET}.s3.amazonaws.com`;
+import {S3SignedUploadResult} from './types';
 
 const ImageContentType = 'image/png';
 const VideoContentType = 'video/mp4';
 
-const s3 = new aws.S3({
-  region: S3_REGION,
-  signatureVersion: 'v4',
+const presigner = new S3RequestPresigner({
+    credentials: fromEnv(),
+    region: S3_REGION,
+    sha256: Hash.bind(null, "sha256"),
 });
 
 interface GetUploadURLParams {
-  key: string;
-  ContentType?: string;
-  Expires?: number;
+    key: string;
+    ContentType?: string;
+    Expires?: number;
 }
 
-export const getUploadUrl = async ({
-  key,
-  ContentType = ImageContentType,
-  Expires = UPLOAD_EXPIRY_SECONDS,
+export const getUrls = async ({
+    key,
+    ContentType = ImageContentType,
 }: GetUploadURLParams): Promise<S3SignedUploadResult> => {
-  const s3Params = {
-    Bucket: S3_BUCKET,
-    Key: key,
-    Expires,
-    ContentType,
-    ACL: S3_ACL,
-  };
+    const url = parseUrl(`${S3_URL}/${S3_BUCKET}/${key}`);
+
+    const signedGetUrlObject = await presigner.presign(
```

Comment:
@p-8-z This one is going to be stored in the DB and used for all subsequent reads. does it have an expiration?